lib/publiccloud/provider.pm

```diff
@@ -213,9 +213,11 @@ Creates an instance on the public cloud provider using ipa command without clean
 =cut
 sub create_instance {
     my ($self, %args) = @_;
+    $args{instance_type} //= get_var('PUBLIC_CLOUD_INSTANCE_TYPE');
```

Comment:
I think we should describe the possible arguments in perlpod.

Reply:
will do so